libs/application/templates/financial-aid/src/lib/formatters.ts

```diff
@@ -159,28 +159,35 @@ export const getStateMessageAndColor: KeyMapping<
 }
 
 export const timelineSections = (
-  created: Date,
-  modified: Date,
+  created?: string,
+  modified?: string,
   showSpouseStep?: boolean,
 ) => {
+  const createdFormatted = created
+    ? format(new Date(created), 'dd/MM/yyyy HH:mm')
+    : null
+  const modifiedFormatted = modified
+    ? format(new Date(modified), 'dd/MM/yyyy HH:mm')
+    : null
```

Comment:
this could be a function, if the format changes, only have to change one place 🤓